### DISCUSSION TITLE
Hello

### DISCUSSION THREAD

**Discussion by ccwower** [community/community#38595] [Other Feature Feedback, Questions, & Ideas]
Hello, I am a student from China, and I hope to work with you to update the hellground project. Now I have encountered some problems, and the VS2012-2015 version cannot successfully compile your core code. Could you please help me solve the problem?

**Comment by ccwower** [comment]
There were several errors during the compilation process and the compilation failed.

Can you change CMAKE to VS2010?

**Comment by airtower-luna** [comment]
It looks like you're looking for help with a software hosted on GitHub. In general you should open an issue *in that repository* in that case, so the developers see it. However I can't find any active repository when searching "hellground" (only ones that are archived, or haven't been updated in a long time), so I'm not sure if there is one that's currently maintained. :sweat_smile: 

However, I don't think anyone would change the build system just so, especially not from a an open source and multi-platform one to a proprietary, Windows-only one. It might be best if you learn how to use [CMake](https://cmake.org/).

**Reply by ccwower** [reply]
https://github.com/ratkosrb/Hellground_core
Can we update and develop together?
